.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- Don't open a PR without follow the workflow of an issue and before a https://petitions.classicpress.net/ -->
```

Comment:
Let's remove this line, please, or move it to a better place (only for certain types of changes, for example).  An issue or a petition is not necessary for all PRs.

Reply:
We had people opening a pr without a issues as example or without a confirm on doing that

Reply:
Yes, I am one of those people, and I will not be opening an issue or a petition before each PR.

If a PR is not something we obviously want, then we will discuss it before merging.

We have already defined the categories of changes that do not require a petition.  Please read this:  https://www.classicpress.net/democracy/#exceptions